omnibus/config/projects/updater-packager.rb

```diff
@@ -0,0 +1,97 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2016-present Datadog, Inc.
+require "./lib/ostools.rb"
+
+name 'updater'
+package_name 'datadog-updater'
+license "Apache-2.0"
+license_file "../LICENSE"
+
+third_party_licenses "../LICENSE-3rdparty.csv"
+
+homepage 'http://www.datadoghq.com'
+
+INSTALL_DIR = '/opt/datadog/updater'
+
+install_dir INSTALL_DIR
+
+if ENV.has_key?("OMNIBUS_WORKERS_OVERRIDE")
+  COMPRESSION_THREADS = ENV["OMNIBUS_WORKERS_OVERRIDE"].to_i
+else
+  COMPRESSION_THREADS = 1
+end
+if ENV.has_key?("DEPLOY_AGENT") && ENV["DEPLOY_AGENT"] == "true"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ Why does `DEPLOY_AGENT` control how much we compress?

Reply:
We don't want to systematically compress to level 9 since it has a relatively heavy cost on the pipeline duration.

The main goal of using the maximum compression level is to save on network costs afterward, and checking if the pipeline is for a deployment is the closest I could come up with without adding a new flag